<launch> 
	<node pkg = "armcar" name = "UltraSonic_Talker_First_Node" type = "UltraSonic_talker.py" output = "screen"> 
		<param name = "my_topic" type = "str" value = "UltraSonic_Talker_First" /> 
		<param name = "GPIO_TRIGGER" type = "int" value = "18" /> 
		<param name = "GPIO_ECHO" type = "int" value = "24" /> 
	</node> 
	<node pkg = "armcar" name = "UltraSonic_Talker_Second_Node" type = "UltraSonic_talker.py" output = "screen"> 
		<param name = "my_topic" type = "str" value = "UltraSonic_Talker_Second" /> 
		<param name = "GPIO_TRIGGER" type = "int" value = "17" /> 
		<param name = "GPIO_ECHO" type = "int" value = "4" /> 
	</node> 
	<node pkg = "armcar" name = "UltraSonic_Talker_Third_Node" type = "UltraSonic_talker.py" output = "screen"> 
		<param name = "my_topic" type = "str" value = "UltraSonic_Talker_Third" /> 
		<param name = "GPIO_TRIGGER" type = "int" value = "22" /> 
		<param name = "GPIO_ECHO" type = "int" value = "27" /> 
	</node> 
</launch> 
